authentication/src/main/java/com/ecommercepractice/authentication/exception/ControllerAdvisor.java

```diff
@@ -0,0 +1,183 @@
+package com.ecommercepractice.authentication.exception;
+
+import com.ecommercepractice.authentication.util.Pair;
+import io.swagger.annotations.ApiResponse;
+import io.swagger.annotations.ApiResponses;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.MethodArgumentNotValidException;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.context.request.WebRequest;
+import org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@ControllerAdvice
+@Slf4j
+public class ControllerAdvisor extends ResponseEntityExceptionHandler {
```

Comment:
Change this class name to express something more according to, like ExceptionHandler or something like that